src/Command/DebugCommand.php

```diff
@@ -46,6 +50,11 @@ protected function configure(): void
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        if ($this->translator instanceof Translator) {
+            $this->translator->setLocale($input->getOption('locale'));
+        } else {
+            $output->write("There is no translator available or configrued, so the locale option will be ignored.\n");
```

Comment:
Not really... would it? I think it will rather fail because the `trans` filter cannot be used in twig if the translator is not available.

Reply:
But it only fails if you want to get the HTML output...